src/util/tr.js

```diff
@@ -499,6 +496,30 @@ const _tr = (msg, {
       const body = "<div class='wzrkPPdscr' style='color:" + textColor + "'>" + descriptionText + '<div></td></tr></table></div>'
       html = css + title + body
     }
+    const properties = {}
+    if (displayObj.preview && displayObj['custom-editor']) {
+      iframe.sandbox = 'allow-scripts allow-popups allow-popups-to-escape-sandbox'
+      /* eslint-disable no-cond-assign */
```

Comment:
we should always avoid using eslint disable unless we absolutely need to

Reply:
Refactored it